keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/Nd4jDoubleTensor.java

```diff
@@ -29,13 +33,27 @@ public static Nd4jDoubleTensor scalar(double scalarValue) {
     }
 
     public static Nd4jDoubleTensor create(double[] values, int[] shape) {
-        return new Nd4jDoubleTensor(values, shape);
+        return new Nd4jDoubleTensor(shape, values);
     }
 
     public static Nd4jDoubleTensor create(double value, int[] shape) {
         return new Nd4jDoubleTensor(Nd4j.valueArrayOf(shape, value));
     }
 
+    public static Nd4jDoubleTensor create(RealMatrix realMatrix) {
```

Comment:
Is this ever used? If it's only used in a test, I'd say just let the user do this if they need it.

Reply:
Good spot, I no longer need this after we merged in your matrix mul. deleted it